docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '2'
+services:
+  db:
+    image: mdillon/postgis
+  web:
+    build:
+      context: .
+      dockerfile: ./dockerfiles/Dockerfile
+    volumes:
+      - .:/opt/streamwebs/data
+    working_dir: /opt/streamwebs
+    command: python /opt/streamwebs/manage.py runserver 0.0.0.0:8000
```

Comment:
Either this line should be `   command: python /opt/streamwebs/data/manage.py runserver 0.0.0.0:8000` or the above `volumes` line should be `     - .:/opt/streamwebs`